### ISSUE TITLE
if I have warp images and matching unwarp images ，how to generate training dataset.

### ISSUE THREAD

**Issue by Liufeiran123** [tanguymagne/UVDoc#26]
if I have warp images and matching unwarp images ，how to generate training dataset.

**Comment by tanguymagne** [comment]
With only the images, it is very hard to obtain the ground truth mapping between the two. 

You could still try to fine-tune our model using the reconstruction loss, which uses only the images, but I doubt it would bring much performance improvement.

A way to obtain the mapping between the two images is to use document image registration methods such as [this one](https://github.com/ZZZHANG-jx/DocAligner), but these methods are never perfectly accurate.

**Comment by Liufeiran123** [comment]
OK，thanks
